### ISSUE TITLE
Test and add new homepage content

### ISSUE THREAD

**Issue by kristinashu** [mozilla/foundation.mozilla.org#5050]
Once code is ready, update the homepage with real content and do a final design QA check (CMS fields, layout, functionality, different devices and browsers...).

### Final designs

https://redpen.io/p/bn82314af4898269e1

### Test on
- [x] homepage-refresh.mofostaging.net
- [x] foundation.mofostaging.net

### Launch on
- [ ] foundation.mozilla.org

### QA copy styles
- [x] Headings: ticket here #5083 
- [x] Body copy
- [x] CTAs

### Gathered Content
- Dynamic content will be determined here https://github.com/mozilla/foundation.mozilla.org/issues/5019
- Evergreen content is [here](https://docs.google.com/document/d/19Y5hNmEU4wix_gCoSvmhMKUk2F1Dht6Nwf1JQAu0lsg/edit#heading=h.jga10kqap7d8)
- Blog profile photos https://github.com/mozilla/foundation.mozilla.org/issues/5020

### Icon JPGs
- [Areas of focus](https://drive.google.com/drive/folders/1nWooXJS-87Z_4gLlFOZgHR3l-KwAC3ba)
- [Partner logos](https://drive.google.com/drive/folders/1jg__Ly6LXXE1S_2PKqjWa6CsayujRgBf?usp=sharing)

### Analytics
- Make sure Analytics (events) are in place: AT RISK - Cade mentioned it may not be possible this sprint (Taís)
- Annotate in GA once site is launched

### Localization
- [Localize homepage](https://github.com/mozilla/foundation.mozilla.org/issues/5093)

### Before launch
- [x] make sure edit rights are/can be granted to folks for the homepage
- [x] make sure evergreen content is revised and in place
- [x] make sure images for non-evergreen content are in place
- [x] make sure content for non-evergreen content is in place. STATUS: ready to go

**Comment by taisdesouzalessa** [comment]
@cadecairos could you please ping me when all features land on https://homepage-refresh.mofostaging.net/en/? I feel that some of the changes haven't landed yet and that is why I am having issues visualizing and editing all fields together in the CMS. Does it make sense? If it doesn't we can chat tomorrow. 

This is my design last QA list for all the features once they land and I can then create implementation tickets if major issues show up (which I think it will be minor since I have been doing QA in individual components).

**Comment by cadecairos** [comment]
Was checking out the new homepage using the responsive design view, and the Hero button extends off the page when the button text is too large:

![image](https://user-images.githubusercontent.com/438003/91073665-6281f680-e609-11ea-99d0-b13e60b4cbd3.png)

l

**Comment by taisdesouzalessa** [comment]
Tks for checking, Cade :). I caught that too and I'll add it to the QA list. Let us know if you find any other issues (the more eyes on QA the better).

**Comment by taisdesouzalessa** [comment]
### Design QA (site front end):
- [ ] DESKTOP/TABLET/MOBILE - Thin black border visible when profile picture has a light background.
<img width="151" alt="image" src="https://user-images.githubusercontent.com/22157921/91075317-baf9c980-e5f2-11ea-8b11-3352f19ec88f.png">

- [ ] DESKTOP/TABLET/MOBILE - "Empower action" icon is smaller than others
<img width="746" alt="image" src="https://user-images.githubusercontent.com/22157921/91075503-08763680-e5f3-11ea-8eff-78de9778bcf1.png">

- [ ] MOBILE: "Our area of focus" title is 24px instead of 22px 
<img width="315" alt="image" src="https://user-images.githubusercontent.com/22157921/91075598-2e9bd680-e5f3-11ea-9a2c-6539f3547d99.png">

- [ ] MOBILE: reduce character counting of CTA to 26 **OR** allow button to break into multiple lines - button is getting cut in smaller screens - as Cade pointed above.
<img width="363" alt="image" src="https://user-images.githubusercontent.com/22157921/91075731-5b4fee00-e5f3-11ea-99b9-9e8ba778d6b2.png">

- [ ] MOBILE (iphone 5 viewport) - top of the title gets cut
<img width="468" alt="image" src="https://user-images.githubusercontent.com/22157921/91075862-8df9e680-e5f3-11ea-87ee-b2d15580087c.png">


### CMS issues:
- [ ] **Not sure this is an issue or it will gets fixed once we merge the update to staging**: when clicking on “view live” on “https://homepage-refresh.mofostaging.net/cms” I am directed to “https://foundation.mofostaging.net/en/”

- [ ] Nice to have: reorder the CMS editing blocks order to follow the live page order.
<img width="664" alt="image" src="https://user-images.githubusercontent.com/22157921/91076051-c4cffc80-e5f3-11ea-8e3b-4186f40b26ff.png">

**Comment by taisdesouzalessa** [comment]
UPDATE: partner logos will change to reflect latest host orgs (I connected with Amy this morning to figure out the logos that would be the best to display). I'll prepare the logos and replace them in the proper folder.

**Comment by Pomax** [comment]
the preview link is a `sites` setting issue, and won't affect the deploy

**Comment by taisdesouzalessa** [comment]
One more thing @sabrinang  identified (tks!!):

- [ ] when we adjust the browser window to narrower width, this is what we see:
<img width="397" alt="image" src="https://user-images.githubusercontent.com/22157921/91103261-efd04580-e61f-11ea-8d4c-09a4e024ed4a.png">

**Comment by taisdesouzalessa** [comment]
Found one more bug - hopefully is the last - in the "quote" section the image doesn't go full width in mobile. Filed a ticket here: #5126 cc: @mmmavis @Pomax

**Comment by taisdesouzalessa** [comment]
And this is launched 🚀 . Thanks, team!
